docs/pages/get-started/react.mdx

```diff
@@ -217,7 +226,7 @@ export const {
     useOthers,
     useUpdateMyPresence, // 👈
   },
-} = createRoomContext(client);
+} = createRoomContext<Presence, Storage, UserMeta>(client);
```

Comment:
Same comment as above. We're starting to explain the `Presence` concept here. `Storage` and `UserMeta` are not defined yet. We should define `Presence` and add a cursor property to it.